include/utils/polygon.h

```diff
@@ -1,5 +1,5 @@
-//Copyright (c) 2022 Ultimaker B.V.
-//CuraEngine is released under the terms of the AGPLv3 or higher.
+// Copyright (c) 2022 Ultimaker B.V.
```

Comment:
```suggestion
// Copyright (c) 2022 UltiMaker B.V.
```

Not sure, but maybe we should start changing these?